src/pages/dashboard/country/[id].page.tsx

```diff
@@ -48,17 +47,66 @@ const Country = () => {
     }
   ];
 
+  const COLUMN_ACTIVE_COUNTRY = [
+    {
+      header: "Project",
+      accessorKey: "project",
+      enableSorting: false
+    },
+    {
+      header: "Trees Planted",
+      accessorKey: "treesPlanted",
+      enableSorting: false
+    },
+    {
+      header: "Hectares",
+      accessorKey: "restoratioHectares",
+      enableSorting: false
+    },
+    {
+      header: "Jobs Created",
+      accessorKey: "jobsCreated",
+      enableSorting: false
+    },
+    {
+      header: "Volunteers",
+      accessorKey: "volunteers",
+      enableSorting: false
+    },
+    {
+      header: "",
+      accessorKey: "link",
+      enableSorting: false,
+      cell: () => {
+        return (
+          <a href="/dashboard/project">
+            <Icon name={IconNames.IC_ARROW_COLLAPSE} className="h-3 w-3 rotate-90 text-darkCustom hover:text-primary" />
+          </a>
+        );
+      }
+    }
+  ];
+
+  const { data: dashboardCountries } = useGetV2DashboardCountries<any>({
+    queryParams: {}
+  });
+  const countrySelected = dashboardCountries?.data.find(
+    (country: any) => country.country_slug === router.asPath.split("/")[3]
```

Comment:
1. The current usage of any can lead to runtime errors and makes it hard to understand the data structure. Instead, define a proper type for dashboardCountries and country. example:
```
interface DashboardCountry {
  country_slug: string;
  // add other properties as needed
}
```
2. Using router.asPath.split("/") to get the country_slug is brittle and could break if the URL structure changes
An option could be router.query to extract route parameters instead of manually splitting the path or inherit from the parents or create context.

Reply:
done